plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.xml

```diff
@@ -61,9 +61,6 @@
         id="com.google.cloud.tools.eclipse"
         name="Google Cloud Platform"
         class="com.google.cloud.tools.eclipse.properties.GooglePropertyPage">
-       <enabledWhen>
-         <reference definitionId="com.google.cloud.tools.eclipse.appengine.onlyInGCPProjects" />
```

Comment:
I understand the `GPE` complaint and I've looked into it whether we could dynamically update the preference page list on the left hand side of the dialog, however we cannot.
So with the risk of annoying some users, I'd rather keep the preferences page always visible to avoid forcing users to close and reopen the dialog in case they add one of the supported facets and want to change any GCP related preferences.